exceptions.ts

```diff
@@ -348,9 +348,9 @@ export class FileError extends OsError {
 
     return new FileError(
       message || err.message,
-      err.code,
+      code || '',
       exitCode,
-      [err.path],
+      [err.path || ''],
```

Comment:
1. This should probably be '<unknown>', like on L322/324
2. Maybe a FileError should allow for zero paths, instead of exactly 1 or 2?

Reply:
I'm going to leave it as 1 for now, since I believe I will always have that path in practice.